app/src/main/java/org/schabi/newpipe/fragments/detail/VideoDetailFragment.java

```diff
@@ -208,6 +208,8 @@
     private Player player;
     private final PlayerHolder playerHolder = PlayerHolder.getInstance();
 
+    private BottomSheetBehavior.BottomSheetCallback bottomSheetCallback;
```

Comment:
Mmmh, I would put this field near `setBottomSheetCallback`, I think it makes more sense

Reply:
Sure, I'll make that change.